data/json/items/armor/helmets.json

```diff
@@ -2289,5 +2606,269 @@
       }
     ],
     "melee_damage": { "bash": 1 }
+  },
+  {
+    "id": "lc_helmet_turban",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "mild steel turban helm" },
+    "description": "A middle eastern helmet, cylindrical on the top with protection for the nose and ears but nothing lower. This one is made in modern times out of mild steel.",
+    "weight": "800g",
```

Comment:
I would advise to adjust weight value for turban helm, too. Just as I said in my previous comment: 1 kg per 1 mm of thickness for total of about 2.5 kg +- 100 g. Otherwise, I approve of the changes and stats.

Reply:
Fixed. Made it 2.4 as its just metal with no leather or anything.